File: Selenium/Project/ProjectActivity4.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.support.ui.WebDriverWait;
import org.testng.Assert;
import org.testng.annotations.Test;

import java.time.Duration;

public class ProjectActivity4 extends BaseTest{
    @Test
    public void popularCourseTest(){

        wait = new WebDriverWait(driver, Duration.ofSeconds(20));
        WebElement firstInfobox = driver.findElement(By.xpath("(//div/h3[@class='entry-title'])[2]"));
        String courseName = firstInfobox.getText();
        Assert.assertEquals("Email Marketing Strategies",courseName);

    }
}

File: PythonActivities/Project_Activity4.py
import time
from selenium import webdriver
from selenium.webdriver.common.by import By

def Project_Activity4():
    driver = webdriver.Chrome()
    driver.get("https://alchemy.hguy.co/lms/")
    courseTitleElement = driver.find_element(By.XPATH,"(//div/h3[@class='entry-title'])[2]").text
    assert courseTitleElement == "Email Marketing Strategies"
    print(courseTitleElement)

Project_Activity4()
